.gitignore

```diff
@@ -16,3 +16,4 @@
 #	src/.deps/
 #	src/Makefile
 #	src/Makefile.in
+/build*/
```

Comment:
Just ```build``` is fine no? Why the trailing slash?

Reply:
I usually use separate build directories for different targets:

```bash
cmake -S. -Bbuild # pc
cmake -S. -Bbuild-rg350 -DCMAKE_TOOLCHAIN_FILE=... # rg350
```

This preserves the build cache for each target and makes my command history more useful

Reply:
The trailing slash is not needed. I like it for clarity when specifying directories though.